aip/0166.md

```diff
@@ -0,0 +1,78 @@
+---
+aip:
+  id: 166
+  state: reviewing
+  created: 2020-06-24
+permalink: /166
+redirect_from:
+  - /0166
+---
+
+# Hierarchically-derived resources
```

Comment:
Typically these are called "policies". I think you are being general here, but would it be worth mentioning the word "policy" so those who are looking at this from the perspective of policies understand that what you discuss applies.

Reply:
This is a great point. Do you think it needs to be in the title, or is the prose sufficient (so it is found in searches and such)?

There are examples of derived resources that are not policies (e.g. the opening hours example that @odsod mentioned) so while I agree with the importance of making it clear that this is useful for policies, I do not want to restrict to them.